docs/source/qvm.rst

```diff
@@ -272,8 +272,8 @@ For more information about creating and adding your own noise models, check out
 Methods
 -------
 
-Now that you have your ``qc``, there's a lot you can do with it. Most users will want to use ``compile``, ``run`` or
-``run_and_measure``, and ``qubits`` very regularly. The general flow of use would look like this:
+Now that you have your ``qc``, there's a lot you can do with it. Most users will want to use ``compile``, ``run``, and
+``qubits`` very regularly. The general flow of use would look like this:
```

Comment:
I don't know how frequently folks use `qc.qubits()` ... certainly not as frequently as `qc.run()` or `qc.compile()`.

Reply:
Sure, will update